devel/playdate-sdk/Portfile

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                playdate-sdk
+version             1.12.2
+revision            0
+categories          devel
+maintainers         @idleberg
+license             Restrictive
+
+description         Playdate SDK
+long_description    This package installs includes a rich set of APIs, a \
+                    compiler, and documentation for Playdate game development.
+
+homepage            https://play.date/dev/
+master_sites        https://download-keycdn.panic.com/playdate_sdk/
+distname            PlaydateSDK-${version}
+distfiles           ${distname}.zip
+
+checksums           rmd160  f0768a68c50b9d92029b562fff36e9235562de80 \
+                    sha256  42c793eb1e5d5251f3f1b8ac0cd1b1489c5f3e5a763b130c31205496d12389ce \
+                    size    30080744
+
+depends_extract     port:xar
```

Comment:
this should likely be `depends_extract-append` as I suspect that the `port:zip` dependency gets added already when setting `use_zip` yes and you don't want to overwrite that